all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -40,29 +40,26 @@ metadata:
     name: jaeger-infra
     jaeger-infra: jaeger-template-all-in-one
 objects:
-- apiVersion: v1
-  kind: DeploymentConfig
+- apiVersion: extensions/v1beta1
```

Comment:
A relatively new version of `oc`, yes. This should be OK, as we require a similar version on the Kubernetes template.